docs/README.md

```diff
@@ -1,5 +1,5 @@
 [![PR validation](https://github.com/dell/csi-baremetal/actions/workflows/pr.yml/badge.svg)](https://github.com/dell/csi-baremetal/actions/workflows/pr.yml)
-[![E2E testing](https://github.com/dell/csi-baremetal/actions/workflows/e2e.yml/badge.svg)](https://github.com/dell/csi-baremetal/actions/workflows/e2e.yml)
+[![E2E testing](https://github.com/dell/csi-baremetal/actions/workflows/e2e.yml/badge.svg?branch=master&event=push)](https://github.com/dell/csi-baremetal/actions/workflows/e2e.yml)
```

Comment:
How will this work for release branches?

Reply:
This works for `master` branch with `push` event. For another branch we need another badge.